events/break-free/index.html

```diff
@@ -72,6 +72,8 @@
             {{ description }}
           </div>
         </script>
+
+  <script type="text/javascript" src="js/action-kit-service.js"></script>

```

Comment:
Do we not need moment.js and icon-service.js here?  Would we need those if we actually used data-actionkit= in this page's script tag?  

If we're not using data-actionkit= in this page, do we still need to include action-kit-service.js?